internal/controller/components/modelcontroller/modelcontroller.go

```diff
@@ -88,6 +88,26 @@ func (s *componentHandler) Init(_ common.Platform) error {
 		return fmt.Errorf("failed to update images on path %s: %w", manifestsPath(), err)
 	}
 
+	if err := odhdeploy.ApplyParams(manifestsPath().String(), "params-cuda.env", imageParamMap); err != nil {
+		return fmt.Errorf("failed to update images on path %s: %w", manifestsPath(), err)
+	}
+	
+	if err := odhdeploy.ApplyParams(manifestsPath().String(), "params-vllm-cpu.env", imageParamMap); err != nil {
+		return fmt.Errorf("failed to update images on path %s: %w", manifestsPath(), err)
+	}
+	
+	if err := odhdeploy.ApplyParams(manifestsPath().String(), "params-vllm-gaudi.env", imageParamMap); err != nil {
+		return fmt.Errorf("failed to update images on path %s: %w", manifestsPath(), err)
+	}
+	
+	if err := odhdeploy.ApplyParams(manifestsPath().String(), "params-vllm-rocm.env", imageParamMap); err != nil {
+		return fmt.Errorf("failed to update images on path %s: %w", manifestsPath(), err)
+	}
+	
+	if err := odhdeploy.ApplyParams(manifestsPath().String(), "params-guardrails-hf-runtime.env", imageParamMap); err != nil {
+		return fmt.Errorf("failed to update images on path %s: %w", manifestsPath(), err)
+	}
+
```

Comment:
Applying substitution on individual file in favor of https://issues.redhat.com/browse/RHOAIENG-33242 to meet rhoai-2.25 target date